requirements.txt

```diff
@@ -1,9 +1,8 @@
-datapackage-pipelines-sourcespec-registry>=0.0.9
+-e git+git://github.com/datahq/planner.git#egg=planner
```

Comment:
It's better to use this syntax:
`pip install https://github.com/user/repository/archive/branch.zip`

See here https://stackoverflow.com/a/24811490/293929